presto-main/src/test/java/com/facebook/presto/cost/TestStatsNormalizer.java

```diff
@@ -73,7 +72,7 @@ public void testDropNonOutputSymbols()
         PlanNodeStatsEstimate estimate = PlanNodeStatsEstimate.builder()
                 .addSymbolStatistics(a, SymbolStatsEstimate.builder().setDistinctValuesCount(20).build())
                 .addSymbolStatistics(b, SymbolStatsEstimate.builder().setDistinctValuesCount(30).build())
-                .addSymbolStatistics(c, SymbolStatsEstimate.buildFrom(UNKNOWN_STATS).build())
+                .addSymbolStatistics(c, SymbolStatsEstimate.buildFrom(SymbolStatsEstimate.unknown()).build())
```

Comment:
`SymbolStatsEstimate.buildFrom(SymbolStatsEstimate.unknown()).build()` should be equivalent to just 
`SymbolStatsEstimate.unknown()`. Please simplify.